ImageClassification/ViewControllers/CameraBoard.swift

```diff
@@ -120,7 +131,32 @@ extension CameraBoard: CameraFeedManagerDelegate {
 	}
 
 	func didOutput(pixelBuffer: CVPixelBuffer) {
-        let currentTimeMs = Date().timeIntervalSince1970 * 1000
+		
+		if frames.count == 13 {
+			print("GOT EM")
+			frames.removeAll()
+		}
+		
+		frames.append(pixelBuffer)
+		print(frames.count)
+		/*
+		                        (*)                  (*)
+		   (__)                  ^                    ^                (__)
+		   (oo)                  |      |             |               @(oo)@
+		   [..]                  |      =             |                [..]@@
+		\ |  U           (-)     |     | |            | (-)             U  @@@@
+		 ||   ==<_\=====/_|______=_____|=|____________=__|____\====/_>==   ||
+		 ||   )  ||||||||||||||||||||||||||||||||||||||||||||||||||||  (   ||
+		 ||___)==||||||||||||||||||||||||||||||||||||||||||||||||||||==(___||
+		 |\====| |||||||||||||||||||||||||||||||||||||||||||||||||||| |====/|
+		 |  \  |     |                                          |     |  /  |
+		 =   * =     =                                          =     = *   =
```

Comment:
@TarekE-dev This is where I'm trying to collect the frames.

Reply:
Also you might wanna checkout [CameraFeedManager](https://github.com/Capstone-Projects-2020-Spring/iASL-iOS/blob/serverModelConnection/ImageClassification/Camera%20Feed/CameraFeedManager.swift) which creates the `didOutput` delegate method.